src/components/LoanCards/Buttons/LendAmountButton.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+	<lend-button
+		price="amountLeft"
+		:loan-id="loanId"
+		@add-to-basket="$emit('add-to-basket', $event)"
+	>
+		Lend ${{ amountLeft }} <span v-if="showNow">now</span>
```

Comment:
Please move the "space" before the `<span>` to be inside the span before " now".

Reply:
Thank yoU!